app/models/form/sales/pages/buyer_previous.rb

```diff
@@ -10,4 +10,10 @@ def questions
       Form::Sales::Questions::BuyerPrevious.new(nil, nil, self, joint_purchase: @joint_purchase),
     ]
   end
+
+  def routed_to?(log, _current_user)
+    return false if log.staircase == 1 && log.form.start_year_after_2024?
```

Comment:
nit: can we make staircase == 1 a method on the log model so it's clearer what value 1 means?